### PR TITLE
Just satisfying my OCD - fixed comment spacing

### PR DESCRIPTION
**Pull request by MartinMuzatko**
**- What I did**

see title

**- How I did it**

with a keyboard

**- How to verify it**

duh.. it is just comments

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

omg. this is a very nice idea to have in the issue template :D
So have my favorite cat:

image

### CODE REVIEW

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Review by vdemeester — APPROVED** [review]
LGTM 🐸 
cc @thaJeztah :D

**Review by thaJeztah — APPROVED** [review]
LGTM 👍

**Comment by yongtang** [comment]
PowerPC Jenkins failure is unrelated. Merging

File: pkg/namesgenerator/names-generator.go [hunk 1]
```diff
@@ -392,10 +392,10 @@ var (
 		// Mary Leakey - British paleoanthropologist who discovered the first fossilized Proconsul skull - https://en.wikipedia.org/wiki/Mary_Leakey
 		"leakey",
 
-		// Henrietta Swan Leavitt - she was an American astronomer who discovered the relation between the luminosity and the period of Cepheid variable stars. https://en.wikipedia.org/wiki/Henrietta_Swan_Leavitt
+		// Henrietta Swan Leavitt - she was an American astronomer who discovered the relation between the luminosity and the period of Cepheid variable stars. https://en.wikipedia.org/wiki/Henrietta_Swan_Leavitt
 		"leavitt",
 
-		//Daniel Lewin -  Mathematician, Akamai co-founder, soldier, 9/11 victim-- Developed optimization techniques for routing traffic on the internet. Died attempting to stop the 9-11 hijackers. https://en.wikipedia.org/wiki/Daniel_Lewin
+		// Daniel Lewin -  Mathematician, Akamai co-founder, soldier, 9/11 victim-- Developed optimization techniques for routing traffic on the internet. Died attempting to stop the 9-11 hijackers. https://en.wikipedia.org/wiki/Daniel_Lewin
```
Comment (meshde) [inline pkg/namesgenerator/names-generator.go]: There are two spaces between *'-'* and *'Mathematician'*. Might wanna remove the extra space there as well. 😛

**Pull request merged**